.babelrc

```diff
@@ -1,3 +1,4 @@
 {
-    "presets": ["metal"]
+    "presets": ["metal", "env"],
+    "plugins": ["syntax-async-functions", "transform-regenerator"]
```

Comment:
Isn't this going to affect all builds we do? I assume this only needs to apply to the a11y bits, so maybe this actually demonstrates that we need to move that to another package...